Sources/QuickTests/Helpers/QuickSpec+QuickSpec_MethodList.m

```diff
@@ -0,0 +1,23 @@
+#import "QuickSpec+QuickSpec_MethodList.h"
+#import <objc/runtime.h>
+
+
+@implementation QuickSpec (QuickSpec_MethodList)
+
++ (NSSet<NSString*> *)allSelectors {
+    id t = [[[self class] alloc] init];
```

Comment:
Two nit-picks: Let's come up with a better variable name than `t` here. How about `specInstance`? Also, we know that this will be an instance of a subclass of `QuickSpec *`, so let's type it as such. In other words, this line should be `QuickSpec *specInstance = ...`.